File: infra/src/main/java/com/beat/infra/external/auth/social/kakao/response/KakaoUserResponse.java
package com.beat.infra.external.auth.social.kakao.response;

import com.fasterxml.jackson.annotation.JsonProperty;

public record KakaoUserResponse(
	@JsonProperty("id")
	Long id,
	@JsonProperty("connected_at")
	String connectedAt,
	@JsonProperty("kakao_account")
	KakaoAccount kakaoAccount
) {
}

File: core/infra/src/main/kotlin/com/beat/infra/external/social/kakao/response/KakaoUserResponse.kt
package com.beat.infra.external.social.kakao.response

import com.fasterxml.jackson.annotation.JsonProperty

internal data class KakaoUserResponse(
    @param:JsonProperty("id")
    val id: Long?,
    @param:JsonProperty("connected_at")
    val connectedAt: String?,
    @param:JsonProperty("kakao_account")
    val kakaoAccount: KakaoAccount?,
)
